deeptime/decomposition/_score.py

```diff
@@ -288,6 +288,9 @@ def vamp_score_cv(fit_fetch: Union[Estimator, Callable], trajs, lagtime=None, n=
     scores = np.empty((n,), float)
     sliding = splitting_mode == 'sliding'
 
+    if random_state is None or isinstance(random_state, int):
+        random_state = np.random.RandomState(random_state)
```

Comment:
I guess the `else` is an instance of `np.random.RandomState`? Maybe safer to raise if it's not?

Reply:
yes!